model.plugin.completion.failure= Le plug-in du modèle {0} n'a pas pu traiter les objets d'entrée.  
